sympy/core/add.py

```diff
@@ -68,7 +68,7 @@ def _unevaluated_Add(*args):
 
 class Add(Expr, AssocOp):
 
-    __slots__ = []
+    __slots__ = ['_constructor_postprocessor_mapping']
```

Comment:
@asmeurer maybe we should have a global dictionary instead of a slot for each instance.